### DISCUSSION TITLE
SPDIFOutput: Max throughput ?

### DISCUSSION THREAD

**Discussion by Efraysse** [pschatzmann/arduino-audio-tools#1139] [Q&A] [answered]
Hello,
I wonder what is the maximum sample rate to be used with SPDIFOutput ?

I set -> **AudioInfo info(128000, 2, 16)** and copy a stream from "http://radiobonheur.ice.infomaniak.ch/radiobonheur-128-1.mp3","audio/mp3" which is suposed to rate 128kbs.

It works but my DAC (Topping E70) stays stuck at 44.1 kbs.

**Comment by pschatzmann** [comment, marked as answer]
I guess you are playing an mp3 which has been recorded at 44100...

**Reply by Efraysse** [reply]
Yes. I confused sample rate and bit rate. The DAC display the sample rate.
Sorry and thanks a lot for your great job.
